instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/jaxrs/JaxrsContextPath.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.jaxrs;
```

Comment:
Can we move it to the javaagent-api module? This class won't be used in library instrumentations after all.